.github/workflows/pr.yaml

```diff
@@ -10,27 +10,41 @@ jobs:
   static-analysis:
     runs-on: macos-15
     steps:
-      - name: Member Check
-        if: ${{ github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER' }}
+      - name: Check Write Permissions
+        uses: octokit/request-action@v2.x
+        id: check_permissions
+        with:
+          route: GET /repos/${{ github.repository }}/collaborators/${{ github.triggering_actor }}/permission
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Debug Permissions Response
+        run: |
+          echo "Permissions raw response: ${{ steps.check_permissions.outputs.data }}"
+      - name: Validate Write Permissions
         run: |
-          echo "Pull Request not triggered by a MSDK team member.  Someone from the team needs to rerun this workflow AFTER it has been deemed safe."
-          exit 1
+          permission=$(echo "${{ fromJson(steps.check_permissions.outputs.data).permission }}")
+          echo "User ${{ github.triggering_actor }} has permissions: $permissions"
```

Comment:
sorry one more I just noticed, I see `$permissions` (plural) here and `$permission` in the lines below, do both variables exist?

Reply:
ah yeah from the latest run I think that needs to be updated
<img width="395" alt="Screenshot 2025-01-24 at 1 12 02 PM" src="https://github.com/user-attachments/assets/76342daa-b736-49e0-b687-341048a9d6aa" />

Reply:
Oops.  Fixing